### PR TITLE
Lt 1159 search goods in goods list

### PR DESCRIPTION
**Pull request by anirbanlahiri-dit**

### CODE REVIEW

File: goods/tests/tests_view.py [hunk 1]
```diff
@@ -43,4 +44,90 @@ def test_fail_view_other_organisations_goods_details(self):
...
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_view_good__query_filter_by_description(self):
+        org = self.test_helper.organisation
+
+        OrgAndUserHelper.create_controlled_good('thing1', org)
+        OrgAndUserHelper.create_controlled_good('Thing2', org)
+        OrgAndUserHelper.create_controlled_good('item3', org)
+
+        url = '/goods/?description=thing'
```
Comment (jfaracik) [inline goods/tests/tests_view.py]: Can you use reverse() please.

File: goods/tests/tests_view.py [hunk 2]
```diff
@@ -43,4 +44,90 @@ def test_fail_view_other_organisations_goods_details(self):
...
+        response = self.client.get(url, **{'HTTP_USER_ID': str(self.test_helper.user.id)})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        response_data = json.loads(response.content)["goods"]
+        self.assertEqual(len(response_data), 2)
+
+        url = '/goods/?description=item'
```
Comment (jfaracik) [inline goods/tests/tests_view.py]: Same as above.

File: goods/tests/tests_view.py [hunk 3]
```diff
@@ -43,4 +44,90 @@ def test_fail_view_other_organisations_goods_details(self):
...
+        response = self.client.get(url, **{'HTTP_USER_ID': str(self.test_helper.user.id)})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        response_data = json.loads(response.content)["goods"]
+        self.assertEqual(len(response_data), 1)
+
+        url = '/goods/'
```
Comment (jfaracik) [inline goods/tests/tests_view.py]: Same as above.

File: goods/tests/tests_view.py [hunk 4]
```diff
@@ -43,4 +44,90 @@ def test_fail_view_other_organisations_goods_details(self):
...
+                            organisation=org)
+
+        Good.objects.create(description='Truck',
+                            is_good_controlled=True,
+                            control_code='ML1',
+                            is_good_end_product=True,
+                            part_number='CL1000',
+                            organisation=org)
+
+        url = '/goods/?part_number=cl'
```
Comment (jfaracik) [inline goods/tests/tests_view.py]: Same as above.

File: goods/tests/tests_view.py [hunk 5]
```diff
@@ -43,4 +44,90 @@ def test_fail_view_other_organisations_goods_details(self):
...
+        response = self.client.get(url, **{'HTTP_USER_ID': str(self.test_helper.user.id)})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        response_data = json.loads(response.content)["goods"]
+        self.assertEqual(len(response_data), 3)
+
+        url = '/goods/?part_number=100'
```
Comment (jfaracik) [inline goods/tests/tests_view.py]: Same as above.

File: goods/tests/tests_view.py [hunk 6]
```diff
@@ -43,4 +44,90 @@ def test_fail_view_other_organisations_goods_details(self):
...
+        response = self.client.get(url, **{'HTTP_USER_ID': str(self.test_helper.user.id)})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        response_data = json.loads(response.content)["goods"]
+        self.assertEqual(len(response_data), 1)
+        self.assertEqual(response_data[0]['description'], 'Truck')
+
+        url = '/goods/?part_number=cl&description=car'
```
Comment (jfaracik) [inline goods/tests/tests_view.py]: Same as above.

File: goods/tests/tests_view.py (hunk 1 above)
Comment (anirbanlahiri-dit) [reply goods/tests/tests_view.py]: Thanks!! I took a shortcut and thought I would fix it later and then forgot :)

Comment (anirbanlahiri-dit) [reply goods/tests/tests_view.py]: fixed now!!

**Pull request merged**